roles/ceph-common/tasks/installs/install_on_debian.yml

```diff
@@ -0,0 +1,43 @@
+---
+- name: install dependencies
+  apt: pkg={{ item }}
+       state=present
+       update_cache=yes
+       cache_valid_time=3600
+  with_items:
+    - python-pycurl
+    - ntp
+    - hdparm
```

Comment:
We normally would define these as a list in the role defaults. Could be a debian_pkgs: list. That way we configure the input at the defaults level, rather than buried in the task itself.